src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -52,7 +52,7 @@ public void postTenant(TenantAttributes tenantAttributes,
           }
 
           vertx.executeBlocking(
-            promise -> new PubSubService(headers, vertx).registerModuleInPubsub(promise),
+            promise -> new PubSubRegistrationService(vertx, headers).registerModuleInPubsub(promise),
```

Comment:
Given that the class name now refers to pub-sub registration, the method could be shortened to `registerModule`

```suggestion
            promise -> new PubSubRegistrationService(vertx, headers).registerModule(promise),
```